scripts/apps/authoring/widgets/widgets.ts

```diff
@@ -140,6 +140,11 @@ export const widgetReactIntegration: IWidgetIntegration = {
     disableWidgetPinning: false,
 };
 
+/**
+ * Used for controlling when a widget is closed intentionally by a user action
+ */
+export let closedOnRender = {closed: false};
```

Comment:
Since the object is only neccessary for mutation to work, it would be better to call the object property "value", because the real name is already used in the constant.

I don't think `closedOnRender` is a good name either. What does it have to do with rendering? `closedIntentionally` sounds better. Also comment on how it may get closed unitentionally.

Reply:
> Also comment on how it may get closed unitentionally.